README.md

```diff
@@ -1,2 +1,22 @@
-# dnanexus_upload_multiqc
-App to upload multiqc files to genomics server
+# Upload Multiqc v1.0
+
+## What does this app do?
+Uploads multiqc HTML files to the viapath genomics server
+
+## What are typical use cases for this app?
+- Upload an HTML file in a one-off instance
+- Upload an HTML file as part of a workflow
+
+## What inputs are required for this app to run?
+An HTML file generated by multiqc
+
+## What does this app output?
+The index.html file replaced on the server by an instance of this application
+
+## How does this app work?
```

Comment:
Please add much more detail here eg:
where it's copied to on the server and how (including integrity). 
That a chronological list of all html reports is created which is used by a python script to fill a template